src/util/PasswordComplexityUtil.js

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2017, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['okta'], function (Okta) {
+
+  var _ = Okta._;
+
+  // return true if the complexity is a number and the value is greater than zero
+  var isComplexityEnabledPosNum = function(complexityValue) {
+    return complexityValue > 0;
```

Comment:
Do you want to also perhaps validate that ``complexityValue`` is indeed a number via something like ``!isNaN(complexityValue)``?